### PR TITLE
At-risk color

### PR DESCRIPTION
**Pull request by gavrielm**
## Description

Do not fill at-risk title with color when only one series present. Can test here: http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22SJLife%22,%22genome%22:%22hg38%22,%22plots%22:[{%22chartType%22:%22cuminc%22,%22settings%22:{%22cuminc%22:{}},%22term%22:{%22id%22:%22Cardiomyopathy%22,%22q%22:{%22breaks%22:[1]}}}],%22nav%22:{%22activeTab%22:1}}

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Comment by airenzp** [comment]
Gavriel @xzhou82  requested me to  allow to edit the color when there is only one serie. Is that what you want to avoid?

**Comment by xzhou82** [comment]
Yes color should be editable for single-series case, and with a method
consistent with multi-series. thus no need to change

**Comment by gavrielm** [comment]
Ok then should we color the entire series at-risk count the same color, as follows:
image

This would be consistent with multi-series case.

**Comment by xzhou82** [comment]
should

**Comment by gavrielm** [comment]
Ok I made entire at-risk legend default color for single series. But it looks like only at-risk legend **title** is clickable for multi-series (for single series, entire at-risk legend is clickable). @airenzp Do you know how to have entire at-risk legend clickable for multiple series?

**Review by airenzp — DISMISSED** [review]
Thanks for the fix Gabriel, it works. I was trying to make the entire row to react to the click but I need more time. We can look at it tomorrow.

**Comment by gavrielm** [comment]
When I click to get cuminc plot of `cardiomyopathy` overlaid with `genetically defined race`, I see this empty white box when clicking legend:
image

It works fine though when using URL: http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22SJLife%22,%22genome%22:%22hg38%22,%22plots%22:[{%22chartType%22:%22cuminc%22,%22settings%22:{%22cuminc%22:{}},%22term%22:{%22id%22:%22Cardiomyopathy%22,%22q%22:{}},%22term2%22:{%22id%22:%22genetic_race%22}}],%22nav%22:{%22activeTab%22:1}}

File: client/plots/cuminc.js [hunk 1]
```diff
@@ -1128,9 +1128,7 @@ function setInteractivity(self) {
 		self.app.tip.hide()
 	}
 
-	self.legendClick = function (event) {
-		event.stopPropagation()
-		const d = event.target.__data__
+	self.legendClick = function (d, x, y) {
```
Comment (gavrielm) [inline client/plots/cuminc.js:1131]: Why are `x` and `y` added as parameters to this function? It doesn't look they are being using in this function.

**Review by airenzp — APPROVED** [review]
We reviewed this together and added all the required fixes

**Pull request merged**